tests/functional/olp-cpp-sdk-dataservice-read/DataserviceReadVolatileLayerClientTest.cpp

```diff
@@ -88,6 +106,105 @@ class VolatileLayerClientTest : public ::testing::Test {
     return CustomParameters::getArgument(kLayerEnvName);
   }
 
+  std::string GetTestTile() {
+    return CustomParameters::getArgument(kTileEnvName);
+  }
+
+  void PrepareForPrefetch() {
+    auto app_id = CustomParameters::getArgument(kPrefetchAppId);
```

Comment:
Also could we use SetUp function for prefetch tests also.

Reply:
Same reason here, prefetch tests require tiles in the layer.
Adding data isn't fast process, so I prefer to make this preparation only for these new tests. What do you think about it?